docs/paper/dev/api/inventory-holder.md

```diff
@@ -0,0 +1,156 @@
+---
+slug: /dev/inventory-holder
+---
+
+# InventoryHolder
+
+Using an InventoryHolder is a great way of identifying your plugin's inventories in events.
```

Comment:
I wouldn't call it "great". I think including those kinds of words doesn't fit in with a doc entry like this. Something like `InventoryHolder can be used to identify your plugins' inventories in events.`